commerce/vtex/transform.ts

```diff
@@ -475,6 +476,15 @@ export const filtersToSearchParams = (
   return searchParams;
 };
 
+export const mapProductClusterIdsToFacets = (collections?: string[]) => {
+  const selectedFacets: SelectedFacet[] = [];
+  collections?.forEach((collection) => {
+    selectedFacets.push({ key: "productClusterIds", value: collection });
+  });
+
+  return selectedFacets;
```

Comment:
Would it be possible to use `map` instead of a `forEach` iteration? This way we can write less code and avoid creating unnecessary variables.

Here's an updated code snippet that uses `map` to generate an array of `SelectedFacet` objects:

```suggestion
  return collections?.map<SelectedFacet[]>((collection) => ({ 
    key: "productClusterIds", value: collection 
  })) ?? [];
```

This code is more concise and easier to read, as it uses a single line arrow function and object shorthand syntax. Additionally, by using `map`, we avoid having to create a separate array variable to store the results of the iteration. Finally, the use of the nullish coalescing operator `??` ensures that an empty array is returned if `collections` is null or undefined.

Reply:
Good catch @gustavo-vasconcellos.